setup.py

```diff
@@ -97,16 +97,21 @@
             "pydivert>=2.0.3,<2.2",
         ],
         'dev': [
+            "awscli>=1.19,<2",
```

Comment:
I have a slight urge to get v2 - but unfortunately they don't publish it on pypi anymore because it's considered "a stand-alone app". v1 is still fully supported, but let's keep a mental note of this.

Also, I just realized that installing CLI tools as Python dependencies feels dirty. Hugo is installed "normally", so why not awscli? https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install

I'll classify twine and tox as edge cases.